analysis_options.yaml

```
@@ -21,6 +21,8 @@ linter:
   # `// ignore_for_file: name_of_lint` syntax on the line or in the file
   # producing the lint.
   rules:
+     - prefer_const_constructors
```

YAMLlint (1.35.1)

[error] wrong indentation: expected 4 but found 5

(indentation)